packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -172,6 +172,16 @@ const CFDsListing = observer(() => {
         return null;
     };
 
+    React.useEffect(() => {
```

Comment:
We need useEffect to set it once only , 

Moved other metrics to this file and setting them in same useEffect based on discussion and suggestion from @habib-deriv